lib/teleterm/clusters/cluster_databases.go

```diff
@@ -44,7 +45,7 @@ type Database struct {
 func (c *Cluster) GetDatabase(ctx context.Context, dbURI string) (*Database, error) {
 	// TODO(ravicious): Fetch a single db instead of filtering the response from GetDatabases.
```

Comment:
It seems we can actually address this todo now that we have `GetDatabases`, right?

Reply:
Yeah, it's coming in a separate PR